app/forms/work_form.rb

```diff
@@ -60,6 +60,7 @@ def sync(*)
             allow_nil: true
   validates 'release', presence: true, inclusion: { in: %w[immediate embargo] }
   validates :keywords, length: { minimum: 1, message: 'Please add at least one keyword.' }
+  validates :contact_email, presence: true, format: { with: Devise.email_regexp }
```

Comment:
Can/should we remove the corresponding model validation now that we have the form validation? Cc: @jcoyne

Reply:
I don't care. This seems reasonable to have in both places.